Docs/Licensure/README.md

```diff
@@ -0,0 +1,10 @@
+# Licensure Resource
+
+The Licensure resource allows for operations regarding a practitioner's licenses.
+
+
+## Operations
+
+| Operation | Description |
+| --------- | ----------- |
+| [Get License Status](GetLicenseStatus.md) | Get License Status allows you to provide a practitioner's federation ID in return for the number of licenses they have. |
```

Comment:
Description should be cased using normal casing.  Use more generic terminology, something like, "Get the license status of a medical professional".
Punctuation is inconsistent. In some tables you put periods and in others you don't. For most description columns you wouldn't. If you need multiple sentences then it does make sense but the entire table would follow.